### ISSUE TITLE
Clean up CLI output

### ISSUE THREAD

**Issue by garris** [garris/BackstopJS#165]
When running the BackstopJS the command line output is noisy.  Some of that is caused by the gulp implementation (see #164) and some of it is just logging out too much stuff at runtime.  It would be a great service to the community if someone could help out with this terminal hygiene issue!   Namaste.

**Comment by levithomason** [comment]
How about adding an actual CLI?  It is strange to have to cd into the node_modules directory and run `npm scripts` on a dependency like this.   Usually, the package.json [defines a bin](https://docs.npmjs.com/files/package.json#bin) property. This `bin` property is what links your dependency into the node_modules/.bin directory, then, users can simply run `backstop` (or whatever you name it) from their own package.json.

We're testing Backstop right now @TechnologyAdvice.  The initial demo went well.  If we stick with this, I'd be willing to contribute to an actual CLI if you're game.

**Comment by garris** [comment]
Yes, absolutely!  A real CLI would be great.  Here are some top level requirements for that...

- maintain backward compatibility with existing npm API
- clean API
- simple implementation -- keep the bar low for other contributions.

That's about it.  I guess users would choose between a global install of BackstopCLI or a local BackstopJS `npm run <command>`-style depending on their needs?

**Comment by levithomason** [comment]
Great.  I'll start a PR and spec out the CLI there in markdown before getting too far in the code.

**Comment by garris** [comment]
Sounds great!  👍

**Comment by joemaller** [comment]
@garris Where's this one at? I started with v2.0 and mostly using browser reporting, but the output seems pretty good to me. 

If you could post something like a marked up screenshot, it would be easier to jump in and help.

**Comment by garris** [comment]
@joemaller sorry about this.  Yes the CLI output was already cleaned up.   In fact many of these bugs have been fixed by version 2 -- I have a list I need to review and close...   I just created an issue to close all the issues that version 2 fixes...

https://github.com/garris/BackstopJS/issues/283

Feel free to review the list and close them out.   My big focus is currently the documentation and blog posts that all need updating.

Cheers!

**Comment by garris** [comment]
Closed by #259
